catalog/solr/docs/src/main/resources/ManContents.adoc

```diff
@@ -1,24 +1,422 @@
-* Works with a standalone Solr Server
+==== Using
 
-== Solr Catalog Provider Implementations
+The Solr Catalog Provider is used in conjunction with an Apache Solr Server data store and acts as the client for an external Solr Server.
+It is meant to be used only with the Standalone Solr Server (`catalog-solr-server`).
 
-=== Implementation Details
+==== Installing and Uninstalling
 
-==== Indexing Text
+===== Prerequisites
 
-When storing fields, the Solr Catalog Provider will analyze and tokenize the text values of STRING_TYPE and XML_TYPE `AttributeTypes`. These types of fields are indexed in at least three ways: in raw form, analyzed with case sensitivity, and analyzed without concern to case sensitivity. Concerning XML, the Solr Catalog Provider will analyze and tokenize XML CDATA sections, XML element text values, and XML attribute values. 
+Before the {branding} Solr Catalog Application can be installed,
 
+* the {branding} Kernel must be running,
+* the {branding} Platform Application must be installed, and
+* the {branding} Catalog Application must be installed.
+
+===== Installing
+
+By default, the {branding} Solr Catalog application installs the External Solr Provider.
+
+===== Uninstalling
+
+[WARNING]
+====
+It is very important to save the KAR file or the feature repository URL for the application prior to an uninstall so that the uninstall can be reverted if necessary.
+====
+
+If the {branding} application is deployed on the {branding} Kernel in a custom installation (or the application has been upgraded previously), i.e., its KAR file is in the  `<INSTALL_DIRECTORY>/deploy` directory, uninstall it by deleting this KAR file.
+
+Otherwise, if the {branding} application is running as part of the {branding} distribution zip, it is uninstalled *the first time and only the first time* using the `features:removeurl` command:
+
+----
+features:removeurl -u <Application's feature repository URL>
+----
+
+.Example
+----
+features:removeurl -u mvn:ddf.catalog/catalog-solr-app/2.3.0/xml/features
+----
+
+The uninstall of the application can be verified by the absence of any of the {branding} application's features in the `features:list` command output.
+
+====== Reverting the Uninstall
+
+If the uninstall of the {branding} application needs to be reverted, this is accomplished by either:
+
+* copying the application's KAR file previously in the `<INSTALL_DIRECTORY>/`deploy directory, OR
+* adding the application's feature repository back into {branding} and installing its main feature, which typically is of the form <applicationName>-app, e.g., `catalog-app`.
+
+.Uninstalling {branding} application from DDF distribution
+----
+features:addurl <Application's feature repository URL>  
+features:install <Application's main feature>
+----
+
+.Example
+----
+ddf@local>features:addurl
+mvn:ddf.catalog.solr/catalog-solr-app/2.3.0/xml/features ddf@local>features:install catalog-solr-external-provider
```

Comment:
Should the shell prompt and command for the features:install be on a new line in the example for reverting an install?